README.md

```diff
@@ -4,6 +4,8 @@
 [![Hackage](https://img.shields.io/hackage/v/exercises.svg?logo=haskell)](https://hackage.haskell.org/package/exercises)
 [![MPL-2.0 license](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](LICENSE)
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/haskell-beginners-2022/exercises)
```

Comment:
I wonder if this link works for forks 🤔 
@nadinda Do you know if it works?

Reply:
Good point. I found this solution on [Gitpod's issue](https://github.com/gitpod-io/gitpod/issues/688), but some browsers don't support it.

Reply:
Interesting issue! It's shame this feature is broken in many browsers 😞 

What do you think about putting this button and link into some existing section, keeping the link as it is but adding a paragraph in a form that you need to edit the URL to smth like `https://gitpod.io/#https://github.com/<my-github-username>/exercises`?